Source/Tests/HarnessHelpers.h

```diff
@@ -161,6 +161,21 @@ namespace FEX::HarnessHelper {
     void Init(std::string const &ConfigFilename) {
       ReadFile(ConfigFilename, &RawConfigFile);
       memcpy(&BaseConfig, RawConfigFile.data(), sizeof(ConfigStructBase));
+      GetEnvironmentOptions();
+    }
+
+    std::vector<std::pair<char const*, char const*>> GetEnvironmentOptions() {
```

Comment:
```suggestion
    std::vector<std::pair<std::string_view, std::string_view>> GetEnvironmentOptions() {
```
Why not return `string_view` here already? Would clean up the implementation a fair bit, since you can avoid the `strlen()` calls then.

Reply:
Resolved.